tests/it/test_dvc.py

```diff
@@ -98,3 +98,9 @@ def test_commit_no_auto_confirm(self, get_choice):
         self.dvc_service.commit(False)
         output = execute_command(["dvc", "dag"], capture_output=True)
         assert "large_file.dvc" in convert_bytes_to_string(output.stdout)
+
+    def test_clone(self):
+        self.fds_service.clone(self.get_remote_url_for_test())
+        self.repo_path = f"{self.repo_path}/hello-world"
+        # Checking dvc pull
+        assert does_file_exist(f"{self.repo_path}/data")
```

Comment:
Why do you need to set `self.repo_path` and not just assert directly?

Reply:
changed